categorize_adjectives/using_kmedoids.py

```diff
@@ -0,0 +1,67 @@
+"""Categorize Adjectives using K-medoids algorithm
```

Comment:
The filename should not ever be "using..". It should just be reflective of what it contains. "kmedoids.py" is fine.